test/fixtures/schema.sql

```diff
@@ -925,6 +925,19 @@ CREATE FUNCTION setprojects(id_l int, id_h int, name text) RETURNS SETOF project
     update test.projects set name = $3 WHERE id >= $1 AND id <= $2 returning *;
 $_$;
 
+-- domains on tables are only supported from pg 11 on
+DO $do$BEGIN
+  IF (SELECT current_setting('server_version_num')::INT >= 110000) THEN
+      CREATE DOMAIN projects_domain AS projects;
```

Comment:
:O I didn't know this was possible. Do you use domains for tables in your projects? I guess that would allow adding additional check constraints to a table.

Reply:
I haven't used those, I think this was just for completeness. Must have stumbled upon those while reading the docs. My use case so far is just domains of "explicit" composite types.